### ISSUE TITLE
Outdated version of can-i-use-light causes build to fail

### ISSUE THREAD

**Issue by benjaminjanic** [madskristensen/WebCompiler#447]
### Installed product versions
- Visual Studio: 2019 (16.3.6)
- This extension: latest (1.12.405)

### Description
If you use this great VisualStudio Nuget Package in the latest version you will get a build error in Visual Studio.
The refrenced tool Browserslist causes the error because it says that caniuse-lite is outdated.

npm update might solve the issue, but not for those users that are new to WebCompiler and don't want to do npm update manually.

### Steps to recreate
1. Try to build a project with WebComplier integrated

### Current behavior
Visual Studio cannot build, because an error is thrown.

### Expected behavior
Info will be logged in Visual Studio's output window that Browslerliste detected an outdated can-i-use-lite version.
It will still build.
![webcompiler-breaks-vs-build](https://user-images.githubusercontent.com/24272663/72601471-c1fe8200-3915-11ea-80b3-e68a36876b18.png)

**Comment by SLOPapa** [comment]
This seems to only happen when "autoPrefix" is true?

**Comment by benjaminjanic** [comment]
Yes that is true, but of course "autoPrefix" is a desired feature as well.
If someone could "just" add the latest versions of Browserslist and caniuse-lite this would fix the issue.

**Comment by RichardGreaves** [comment]
This is causing issues for me, too.  I really need to be able to use autoPrefix with my SASS files.

**Comment by DarthSonic** [comment]
How can I update using package installer? Error is 'npm' is not recognized as an internal or external command,

**Comment by tulbox** [comment]
Just put the following into a powershell script (deletes the problematic node_module packages and then reinstalls them):

```
## Remove current version of caniuse-lite & browserlist
rm -r -fo $Home\AppData\Local\Temp\WebCompiler*\node_modules\caniuse-lite\
rm -r -fo $Home\AppData\Local\Temp\WebCompiler*\node_modules\browserlist\

## Install caniuse-lite and browserlist new versions in proper directory
cd $Home\AppData\Local\Temp\WebCompiler*
npm i caniuse-lite browserlist

## Popup to confirm installation completed
$wshell = New-Object -ComObject Wscript.Shell
$wshell.Popup("caniuse-lite & browserlist have been updated",0,"NPM Install Complete",0x0 + 0x40 + 0x1000)
```
